### DISCUSSION TITLE
Deye 6k G03 works with 1.4.3 - failed after update to v1.5.0

### DISCUSSION THREAD

**Discussion by pawelsn** [StephanJoubert/home_assistant_solarman#387] [General]
Colleagues,

Solarman 1.4.3 was working fine with Deye 6k G03.
No updates anymore after update to v1.5.0

With v1.5.0 I have noticed in the logs:
`2023-09-06 10:29:25.170 WARNING (SyncWorker_7) [custom_components.solarman.solarman] Querying [3 - 112] failed, [0] retry attempts left`
`2023-09-06 10:29:25.171 WARNING (SyncWorker_7) [custom_components.solarman.solarman] Querying registers [3 - 112] failed, aborting.`
`2023-09-06 10:29:53.880 WARNING (SyncWorker_1) [custom_components.solarman.solarman] Querying [3 - 112] failed with exception [error: unpack requires a buffer of 220 bytes]`

I use "deye_string.yaml" for Deye SUN-6k-G03 device
Integration started to work again after downgrade to 1.4.3

**Comment by Pablo-or-Pablo** [comment]
I have the similar problem with Three phase LV Hybrid 12kW - it is ok for 1 minute, and then is disconnected, nad in next 1 minute is connected:

Logger: custom_components.solarman.solarman
Source: custom_components/solarman/solarman.py:61
Integration: Solarman ([documentation](https://github.com/StephanJoubert/home_assistant_solarman/blob/main/README.md), [issues](https://github.com/StephanJoubert/home_assistant_solarman/issues))
First occurred: 10:43:42 (286 occurrences)
Last logged: 11:26:35

Querying registers [672 - 679] failed, aborting.
Querying [644 - 653] failed with exception [V5FrameError: V5 frame contains invalid sequence number]
Querying [644 - 653] failed, [1] retry attempts left
Querying [644 - 653] failed, [0] retry attempts left
Querying registers [644 - 653] failed, aborting.

With previous version everything was ok.

**Reply by Dummy0815** [reply]
I think I found an solution. For Details see https://github.com/StephanJoubert/home_assistant_solarman/discussions/271#discussioncomment-7158802

**Reply by techface10** [reply]
@Dummy0815 this solution is also not working. After installing this it stopped reading from hybrid inverter also. Something they have changed in 1.5.0 caused the issue string.yaml is working fine in 1.4.3

**Comment by techface10** [comment]
I am also facing the same issue with my Sun 10K G03 using deye_string.yaml. After downgrading, it is working fine, but most of the parameters are updating once every 5 minutes, not in real-time. Is there any way to make these parameters update in real-time? I have a 5kW hybrid system in the same house, and it is working flawlessly with deye_hybrid.yaml, updating in real-time.

**Comment by Torolex242** [comment]
I also have a zuchetti HYD6000 inverter and it does not display sensor changes through solarman. I've been trying to do it for a few days, I've seen it work with other inverters selected from solarman's "configuration" but it doesn't send the data well, it breaks down.Has anyone else had this problem? Can he tell me how I can fix it?
